### ISSUE TITLE
fine-tuning tricks

### ISSUE THREAD

**Issue by ToUranus** [Time-MoE/Time-MoE#53]
Hello, excellent work. I attempted to fine-tune on the power load forecasting dataset in the new energy field. I discovered that when comparing the Time-Moe large model with and without fine-tuning using load data, the improvement of the fine-tuned model was very limited, and even lower than the baseline. The prediction length is 288 points, with one point every 5 minutes, for a one-day-ahead prediction. The length of historical data is 7 days, that is, 7 * 288. I used a learning rate of 5e-6 and trained for 500 epochs. Is there a problem with the fine-tuning? And are there any useful tricks? Thank you for your reply

**Comment by Maple728** [comment]
Thank you for your interest in our work.

The default `max_seq_length` is set to `1024`. However, in your case, the sequence length extends to $7 \times 288 + 288 = 2304$. Therefore, increasing `max_seq_length` to `3096` might be beneficial. Additionally, if you're experiencing issues with the number of epochs during fine-tuning, you may find it helpful to reduce them.

Please feel free to contact us, if you have any questions.

**Comment by ToUranus** [comment]
Thank you for your reply. In the project cases you provided, when testing data that is not standardized, zero - normalization is carried out first. So, should normalization also be performed during fine - tuning? Since my data is a non - negative dataset, it may be affected if zero - normalization is used. Do the fine - tuning model and the testing model need to maintain consistency in normalization?

**Comment by ToUranus** [comment]
And, should the test set use the normalization parameters of the training set? Thank you for your reply.

**Comment by Maple728** [comment]
> Thank you for your reply. In the project cases you provided, when testing data that is not standardized, zero - normalization is carried out first. So, should normalization also be performed during fine - tuning? Since my data is a non - negative dataset, it may be affected if zero - normalization is used. Do the fine - tuning model and the testing model need to maintain consistency in normalization?

We recommend applying normalization to the sequence before inputting it into the model. Consistently using zero normalization during both training and forecasting is considered the best practice.

P.S. Zero normalization does not significantly negatively impact non-negative datasets.

**Comment by Maple728** [comment]
> And, should the test set use the normalization parameters of the training set? Thank you for your reply.

It depends on your scenario. If you aim to obtain comparable metrics for different context lengths, you should use the sequence statistics (mean and standard deviation) derived from the training set. However, if your goal is to make predictions in  situations where it's challenging to maintain the sequence statistics from the training set, normalizing the sequence values within the context can be beneficial.
